app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt

```diff
@@ -80,13 +80,13 @@ object ArchiveParser {
         // TODO: Check more errors
     }
 
-    class Archive(
-        val res: String,
-        val name: String,
-        val size: String,
-        val cost: String,
-        val isHAtH: Boolean,
-    )
-
-    class Result(val paramOr: String, val archiveList: List<Archive>, var funds: HomeParser.Funds?)
+    class Result(val paramOr: String, val archiveList: List<Archive>, var funds: Funds?)
 }
+
```

Comment:
`@Stable` here.

Reply:
Isn't it stable anyway since all its parameters are immutable primitives.

Reply:
Isn't it stable anyway since all its parameters are immutable primitives?

Reply:
Yeah, you are right.
https://github.com/androidx/androidx/blob/c1540efeacabdf79ba99ca53268332794cfd481a/compose/compiler/compiler-hosted/src/main/java/androidx/compose/compiler/plugins/kotlin/analysis/Stability.kt#L143